exercises/01.principles/01.problem.intentions/README.mdx

```diff
@@ -1,4 +1,6 @@
-# Intentions
+# Write an Intentional Test
```

Comment:
Does this read right? 

I read it as "a test that is written with intention (i.e. purpose). But the original meaning of the exercise is more like "write a test _for_ the intention (behind your code). 

This may be a language barrier issue though.

Reply:
I like to keep these titles short

Reply:
I think of exercise titles as chapter titles in a book. They should summarize what's going on, hint at important aspects, give an idea about what lies ahead, all while being short and, ideally, memorable. 

But I'm not doing this workshop in isolation, and having feedback from folks who know far more than I do is why I'm here. For example, I'd love to learn how the rest of you think of the difference between these titles. What didn't work before, what works better now.

Reply:
For me I'm just thinking about how it'll look in the UI, and longer titles don't look as good 😅